src/components/error/ErrorBoundary.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import * as React from 'react';
+import Raven from 'raven-js';
+import BlueScreen from '.';
+
+type State = {
+  error: ?any,
+};
+
+type Props = {
+  children: any,
+  fallbackComponent?: ?any,
+};
+
+class ErrorBoundary extends React.Component<Props, State> {
+  state = { error: null };
+
+  componentDidCatch = (error: any, errorInfo: any) => {
+    this.setState({ error });
+    Raven.captureException(error, { extra: errorInfo });
+  };
+
+  render() {
+    const { error } = this.state;
+    const { fallbackComponent: FallbackComponent, children } = this.props;
+
+    if (error) {
+      if (this.props.fallbackComponent) {
+        // $FlowFixMe
+        return <FallbackComponent />;
```

Comment:
We have a component like this already, no need to create your own: `import ErrorBoundary from 'react-error-boundary'`

Literally does the exact same thing 😅 Docs: https://github.com/bvaughn/react-error-boundary